lightcurve_fitting/lightcurve.py

```diff
@@ -427,6 +427,7 @@ def calcPhase(self, rdsp=False):
                 raise Exception('must run lc.findPeak() first')
             elif rdsp:
                 self.meta['refmjd'] = self.meta['peakdate']
+                warnings.warn('refmjd and explosion not in meta. Setting phase based on peakdate')
```

Comment:
Technically `'explosion'` could be in `meta` in this case. It doesn't check.

You can only get to this line by explicitly calling `calcPhase(rdsp=True)` so I'm wondering if this warning might not be necessary.

Reply:
I think I wanted to alert the user that `peakdate` was being used. The case that prompted me to write this was I was putting `refdate` instead of `refmjd` in the meta and the code was happily defaulting to something else without telling me and so I had no idea that I had the meta wrong. But I'd be happy to remove the explosion from the warning.